js/docs/src/content/docs/video-metadata/signing.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "How Signing Works"
+sidebar:
+  order: 20
+---
+
+The signing process integrates the user's identity and preferences with the C2PA
+standard to produce a verifiable stream. At a high level, this is how it works:
+
+1. **Key Generation**: The user clicks "Generate Stream Key" on the Streamplace
+   frontend to create a `secp256k1` keypair.
+2. **Key Distribution**: The user is given a stream key that includes the
+   private key combined with their DID, encoded in a multibase format. The
+   corresponding public key is stored in the user's PDS as a `place.stream.key`
+   AT Protocol record for public verification.
+3. **Node Synchronization (Key)**: When the `place.stream.key` record is
+   created, the AT Protocol firehose picks it up. Streamplace nodes then sync
+   this record to a local SQLite database (typically
+   `~/.streamplace/db.sqlite`).
```

Comment:
Not true anymore, it's in `~/.streamplace/index/index_2.sqlite`, and that `2` gets bumped any time our data model changes and we reindex. But I don't think it's relevant detail, we can probably just cut it
```suggestion
   this record to a local SQLite database.
```